db.json

```diff
@@ -1379,7 +1425,7 @@
   "application/rdf+xml": {
     "source": "iana",
     "compressible": true,
-    "extensions": ["rdf","owl"]
+    "extensions": ["rdf","rdfs","owl"]
   },
```

Comment:
.rdfs is not mentioned at https://www.iana.org/assignments/media-types/application/rdf+xml

Reply:
It was added at https://gitlab.freedesktop.org/xdg/shared-mime-info/-/commit/150637a3c0ec0b4f321bd49dc41650b8e71c559f - ``Closes: #8389`` refers to https://bugs.freedesktop.org/show_bug.cgi?id=8389 , which is thankfully still operational.  So in addition to commit messages, the background which led to each commit is likely also available.

fwiw, .rdfs is RDF Schema , and it is expressed as a RDF, and can be serialised as XML, so this addition is fine.